torch/_dynamo/variables/tensor.py

```diff
@@ -1039,3 +1072,47 @@ def call_function(
             )
 
         return super().call_function(tx, args, kwargs)
+
+
+class TypedStorageVariable(VariableTracker):
+    def __init__(self, value, **kwargs):
+        self.value = value
+        super().__init__(**kwargs)
+
+    def reconstruct(self, codegen):
+        return super().reconstruct(codegen)
+
+    def as_proxy(self):
+        return self.proxy
+
+    def var_getattr(self, tx, name: str) -> VariableTracker:
+        if name == "_size":
+            return variables.LambdaVariable(
+                lambda *args, **kwargs: self.call_method(tx, name, [], {})
+            ).add_options(self)
+        return super().var_getattr(tx, name)
+
+    def call_method(
+        self,
+        tx,
+        name,
+        args: "List[VariableTracker]",
+        kwargs: "Dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        if name == "_data_ptr":
+            return ConstantVariable(self.value._data_ptr())
```

Comment:
This is exceedingly unlikely to be right, as you are now burning in pointer constants into the traced Dynamo graph. But typically, I would expect storages to have "tensor" nature and be varying across runs.

Reply:
Or if value is a fake storage, I would expect this to always be a null pointer, which would also be wrong to burn in as a constant variable!

Reply:
Agreed, I wonder if we should put storage objects in the graph...